specs/registries/format.json

```diff
@@ -0,0 +1,375 @@
+        "types": ["string"],
+        "examples": ["^[a-zA-Z0-9]+$"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "relative-json-pointer": {
+        "description": "A JSON string representation of a relative JSON Pointer as defined in draft RFC 01",
+        "definingBody": "JSON Schema",
+        "definition": "https://json-schema.org/draft/2020-12/json-schema-validation.html#name-json-pointers",
+        "types": ["string"],
+        "examples": ["1/0"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-binary": {
+        "description": "Structured fields byte sequence as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-byte-sequences",
+        "types": ["string"],
+        "examples": ["U3dhZ2dlciByb2Nrcw=="],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-boolean": {
+        "description": "Structured fields boolean as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-booleans",
+        "types": ["string"],
+        "examples": ["true", "false"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-decimal": {
+        "description": "Structured fields decimal as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-decimals",
+        "types": ["number"],
+        "examples": ["123.45"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-integer": {
+        "description": "Structured fields integer as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-integers",
+        "types": ["number"],
+        "examples": [123],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-string": {
+        "description": "Structured fields string as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-strings",
+        "types": ["string"],
+        "examples": ["example"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "sf-token": {
+        "description": "Structured fields token as defined in RFC8941",
+        "definingBody": "RFC 8941",
+        "definition": "https://www.rfc-editor.org/rfc/rfc8941#name-tokens",
+        "types": ["string"],
+        "examples": ["token"],
+        "deprecated": false,
+        "supportedBy": []
+    },
+    "time": {
+        "description": "Time as defined by full-time - RFC3339",
+        "definingBody": "JSON Schema",
+        "definition": "https://json-schema.org/draft/2020-12/json-schema-validation.html#name-dates-times-and-duration",
+        "types": ["string"],
+        "examples": ["13:45:30"],
```

Comment:
https://www.rfc-editor.org/rfc/rfc3339.html#section-5.6 defines `full-time` as:
```
time-numoffset  = ("+" / "-") time-hour ":" time-minute
time-offset     = "Z" / time-numoffset

full-time       = partial-time time-offset
```

`time-offset` is non-optional and non-empty, so I don't think `"13:45:30"` is valid in this scenario - it would need to be something like `"13:45:30Z"` or `"13:45:30+01:00"`